README.md

```diff
@@ -25,7 +25,7 @@ NGINX Gateway Fabric uses [NGINX Agent](https://github.com/nginx/agent) to confi
 ## Getting Started
 
 1. [Get started using a kind cluster](https://docs.nginx.com/nginx-gateway-fabric/get-started/).
-2. [Install](https://docs.nginx.com/nginx-gateway-fabric/installation/) NGINX Gateway Fabric.
+2. [Install](https://docs.nginx.com/nginx-gateway-fabric/install/) NGINX Gateway Fabric.
 3. Deploy various [examples](examples).
 4. Read our [How-to guides](https://docs.nginx.com/nginx-gateway-fabric/how-to/).
```

Comment:
I think this one needs to be updated since you moved to `Traffic Management` and `Traffic Security`.

Reply:
Line 4, the initial intention was to link to all of our how-to guides, which are now spread out a bit more.

Reply:
Would a general link to the root of documentation work instead? 

Otherwise this list could end up becoming a manual recreation of the entire documentation sitemap, aside from the link to the example folder.

Reply:
Yeah that's probably fine. And maybe mention that's where they can view how to configure, secure, monitor their traffic. Something along those lines.